SPECS/rubygem-test-unit/rubygem-test-unit.spec

```
@@ -0,0 +1,55 @@
+%global gem_name test-unit
+Summary:        An xUnit family unit testing framework for Ruby
+Name:           rubygem-test-unit
```

Just for the sake of consistency with other specs:
```suggestion
Name:           rubygem-%{gem_name}
```